rust/src/sdp/logger.rs

```diff
@@ -102,6 +102,9 @@ fn log_media_description(
             if let Some(conn_data) = &m.connection_data {
                 log_connection_data(conn_data, js)?;
             }
+            if let Some(enc_key) = &m.encryption_key {
```

Comment:
Maybe if you make `pub struct SdpMessage` to `pub(crate) struct SdpMessage` you would have got a warning that this field was unused ;-)